deploy/demo/template.json

```diff
@@ -363,6 +363,60 @@
              "metadata": {
              "description": "Specifies the location in which to create the workspace."
              }
+        },
+        "appInsightsName": {
+             "type": "string",
+             "defaultValue": "[concat('appinsights-', take(uniqueString(subscription().subscriptionId, resourceGroup().id, parameters('solutionName')), 5))]",
+             "metadata": {
+                  "description": "Specifies the name of application insights."
+             }
+        },
+        "appInsightsType": {
+             "type": "string",
+             "defaultValue": "web",
+             "allowedValues": [
+                    "web",
```

Comment:
Is this even needed?   What would be the outcome of changing this?

Reply:
Removed it, tested without this and it works